src/web/api/netdata-swagger.yaml

```diff
@@ -2561,33 +2561,33 @@ components:
     api:
       description: The version of the API used.
       type: integer
-    agents:
+    Agents:
```

Comment:
This looks wrong. I suggest skipping blind update of swagget files.

Reply:
hey, true, it was not blind, I just missed it in the review process. I will look for such changes